packages/nextjs/services/database/bg-imports/0002_batch_data.ts

```diff
@@ -48,6 +48,7 @@ async function importData() {
         contractAddress: batch.contractAddress,
         telegramLink: batch.telegramLink,
         bgSubdomain: `batch${batch.name}`,
+        network: batch.network as BatchNetwork,
```

Comment:
note: there's no network field in bg api, so this is just for avoiding type errors?